frigate/detectors/detector_config.py

```diff
@@ -51,6 +51,9 @@ class ModelConfig(BaseModel):
     model_type: ModelTypeEnum = Field(
         default=ModelTypeEnum.ssd, title="Object Detection Model Type"
     )
+    order: list[int] = Field(
```

Comment:
I think the name should be more descriptive like `model_output_tensor_order` or something